ros2cli_common_extension/package.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2cli_common_extensions</name>
+  <version>0.1.0</version>
+  <description>Meta package for ros2cli common extensions</description>
+  <maintainer email="ivanpauno@ekumenlabs.com">Ivan Santiago Paunovic</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>ros2action</exec_depend>
+  <exec_depend>ros2cli</exec_depend>
+  <exec_depend>ros2component</exec_depend>
+  <exec_depend>ros2doctor</exec_depend>
+  <exec_depend>ros2interface</exec_depend>
+  <exec_depend>ros2lifecycle</exec_depend>
+  <exec_depend>ros2multicast</exec_depend>
+  <exec_depend>ros2node</exec_depend>
+  <exec_depend>ros2param</exec_depend>
+  <exec_depend>ros2pkg</exec_depend>
+  <exec_depend>ros2run</exec_depend>
+  <exec_depend>ros2service</exec_depend>
+  <exec_depend>ros2topic</exec_depend>
+  <exec_depend>sros2</exec_depend>
+  <exec_depend>ros2launch</exec_depend>
```

Comment:
It would be great to add launch_xml and launch_yaml to the list.
Without them, `ros2 launch` only has support for python launch file.